bits/sendgrid/sendgrid.go

```diff
@@ -0,0 +1,102 @@
+package sendgrid
+
+import (
+	"context"
+	"net/http"
+
+	"encore.dev/beta/errs"
+	"encore.dev/pubsub"
+	"encore.dev/rlog"
+	"github.com/sendgrid/sendgrid-go"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+)
+
+// Encore docs about secrets: https://encore.dev/docs/primitives/secrets
+var secrets struct {
+	SendGridAPIKey string
+}
+
+type SendParams struct {
+	From struct {
+		Name    string `json:"name,omitempty"`
+		Address string `json:"address"`
+	} `json:"from"`
+	To struct {
+		Name    string `json:"name,omitempty"`
+		Address string `json:"address"`
+	} `json:"to"`
+	Subject string `json:"subject"`
+	Text    string `json:"text"`
+	Html    string `json:"html"`
+}
+
+type SendResponse struct {
+	MessageID string `json:"message_id"`
+}
+
+// Send sends an email using the SendGrid API.
+// https://docs.sendgrid.com/api-reference/mail-send/mail-send
+//
+//encore:api public method=POST path=/sendgrid
```

Comment:
This should be a private method, since we don't want it to be called by the outside world

Reply:
Thanks, fixed 👍 
5162819bdc7367db5cd47e03f4706a2f4fed5184